### ISSUE TITLE
Open up Registries to IG, TAG, AB

### ISSUE THREAD

**Issue by frivoal** [w3c/process#902]
Currently, I believe only Working Groups can issue registries. Given that Registries do not contain normative statements and are not subject to the patent policy, I wonder if we could also allow IGs, the AB, and TAG to issue registries as well. I don't really see any reason to restrict them to WGs.

The Process changes to achieve that would be minimal.

For instance, I was wondering if the list of reusable boilerplate text for charters (as discussed in https://github.com/w3c/process/issues/425) could be an AB-maintained repo.

**Comment by dwsinger** [comment]
I don't see how, as the _definition_ of a registry _is_ confined to a rec and hence WGs. But anyone can then _operate_ a registry.

I don't see repos of useful stuff as being a Registry in the formal sense.

**Comment by frivoal** [comment]
> [Registries](https://www.w3.org/policies/process/#registry) can be published either as a stand-alone [technical report](https://www.w3.org/policies/process/#technical-report) on the [Registry Track](https://www.w3.org/policies/process/#registry-track) called a registry report, or incorporated as part of a [Recommendation](https://www.w3.org/policies/process/#RecsW3C) as a registry section.

**Comment by frivoal** [comment]
The registry definition is handled in a way that is similar to a REC, and can be hosted in a REC, but doesn't have to.
https://www.w3.org/policies/process/#reg-pub

**Comment by dwsinger** [comment]
I am sure we decided that Registry Definitions must be in a Technical Report; indeed that seems cited but not stated in the Process.

**Comment by dwsinger** [comment]
We wanted AC approval of the definition of a Registry

**Comment by frivoal** [comment]
> I don't see how, as the definition of a registry is confined to a rec and hence WGs.

> I am sure we decided that Registry Definitions must be in a Technical Report

> We wanted AC approval of the definition of a Registry

Yes, the definitions must be in a technical report, and yes the AC does get to approve the definitions of a registry. However this technical report can *either* be a REC (which get us the requirement for AC review) *or* use the "registry report" type of technical report, created for that purpose, which has equivalent requirements, including the AC review. Some early iterations of the registry process did not have this alternative, but the one we landed on does. I

So, while we cannot enable groups other than WGs to do Registries-in-a-REC since only WGs can do RECs, we could enable other groups to do Registries-in-a-Registry-Report.

> I don't see repos of useful stuff as being a Registry in the formal sense.

It'd be narrower than that:
repos of useful stuff, maintained according to pre-defined rules which can only be changed with the approval of the AC.

**Comment by jyasskin** [comment]
I suspect this isn't needed.

Registries provide an easier way to add things to Recommendations, when full W3C or even Group consensus isn't needed to add a certain class of things. IGs, the TAG, and the AB don't usually publish documents with that level of review, although Statements are now possible. For anything less than a Statement, and especially for TAG and AB documents, I suspect updates to any registry-like list should just get the group's consensus and have the whole document republished as a Note. Boilerplate charter text, in particular, seems like it should get AB review for updates, and not anything lighter-weight.

If a Statement arises in which it'd be useful to have a registry, I'm not opposed to letting it use one, but I feel like that decision should be made knowing the concrete use case.

**Comment by nigelmegitt** [comment]
I think it'd be reasonable to allow an IG or even a CG to have a role in managing the contents of a Registry Table, if a WG is willing to permit them to, in the Registry Definition. I also would like to know the use case for an IG/TAG/AB defining a Registry.

**Comment by dwsinger** [comment]
1. defining a registry. Has to be in a Rec, so we're in WGs.
2. operating a registry. absolutely, the registry definition can identify any person, group, dog or other domestic or non-domestic competent sentient being as the registry operator.

which one are we talking about?

**Comment by frivoal** [comment]
@dwsinger I opened this issue about the first one. But go check the process (in particular "registry report" in https://www.w3.org/policies/process/#reg-pub). "has to be in a REC" is an oversimplification. That was a proposal at some point, but what we landed on isn't exactly that, and what we have could work in an IG (or AB, or TAG) if we wanted to.

on the second point, I agree that it's already covered.

**Comment by frivoal** [comment]
@nigelmegitt , as I mentioned in the opening comment, here's a use case:

> For instance, I was wondering if the list of reusable boilerplate text for charters (as discussed in https://github.com/w3c/process/issues/425) could be an AB-maintained repo.

**Comment by fantasai** [comment]
@nigelmegitt For IGs, for example, the Internationalization IG could maintain a registry of text metrics per writing system.

**Comment by fantasai** [comment]
Wrt Florian's example, it's particularly interesting because it means the AB sets the rules for adding new boilerplates, but the Team can be responsible for maintaining the registry on its own.

**Comment by jyasskin** [comment]
Which registry are you thinking of in #904? https://www.w3.org/TR/i18n-glossary/ is maintained by the Internationalization WG, so it can be a normal registry.

**Comment by nigelmegitt** [comment]
These use cases suggest to me that we should draw a stronger line between Registries and other "lists of interesting stuff". The point of a Registry is that it is important that its values are managed, and that they are reliable, under some kind of change control, so that they can be used with a known level of stability. For example, this makes them useful for defining value sets that can be referenced in Recs. Lists

In the case of Charter sections, if we were to restructure Charters so they pulled in statements by reference instead of by copying them, then it would be important that they are indeed managed carefully. (aside: I think this is probably a bad idea in this particular case)

I would argue that IGs must not be able to publish Registry Definitions but that TAG and AB may, because of the level of "formality" of those groups: in my view it a Registry Definition needs to be considered as substantive as a Rec, so groups that cannot publish Recs cannot publish Registry Definitions.

**Comment by frivoal** [comment]
> I would argue that IGs must not be able to publish Registry Definitions but that TAG and AB may, because of the level of "formality" of those groups: in my view it a Registry Definition needs to be considered as substantive as a Rec, so groups that cannot publish Recs cannot publish Registry Definitions.

IGs are able to make W3C statements, which have the same level of scrutiny as RECs, except for implementation requirements. Given that, I don't see what should stop them from making registries. They'll have to abide by the same rules and level of formality as anyone else while doing so.

> In the case of Charter sections, if we were to restructure Charters so they pulled in statements by reference instead of by copying them, then it would be important that they are indeed managed carefully. (aside: I think this is probably a bad idea in this particular case)

That is indeed the motivating use case for me. I am curious as to why you think that's a bad idea, though we should discuss that over in https://github.com/w3c/process/issues/425
